common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -292,21 +293,6 @@ def get_html(self):
         transcripts = self.get_transcripts_info(include_val_transcripts=feature_enabled)
         track_url, transcript_language, sorted_languages = self.get_transcripts_for_student(transcripts=transcripts)
 
-        # CDN_VIDEO_URLS is only to be used here and will be deleted
-        # TODO(ali@edx.org): Delete this after the CDN experiment has completed.
-        html_id = self.location.html_id()
-        if self.system.user_location == 'CN' and \
-                settings.FEATURES.get('ENABLE_VIDEO_BEACON', False) and \
```

Comment:
We may want to do this as a separate PR because the VIDEO_BEACON settings can probably also be removed.